### DISCUSSION TITLE
Welcome to theashishgavade Discussions!

### DISCUSSION THREAD

**Discussion by theashishgavade** [theashishgavade/theashishgavade#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

```

█████████████████████████████████████████████████████████████████████████████████████████
█░░░░░░░░░░░░░░█░░░░░░░░░░░░░░█░░░░░░██░░░░░░░░████░░░░░░██████████░░░░░░█░░░░░░░░░░░░░░█
█░░▄▀▄▀▄▀▄▀▄▀░░█░░▄▀▄▀▄▀▄▀▄▀░░█░░▄▀░░██░░▄▀▄▀░░████░░▄▀░░░░░░░░░░░░░░▄▀░░█░░▄▀▄▀▄▀▄▀▄▀░░█
█░░▄▀░░░░░░▄▀░░█░░▄▀░░░░░░░░░░█░░▄▀░░█░░░▄▀░░░░████░░▄▀▄▀▄▀▄▀▄▀▄▀▄▀▄▀▄▀░░█░░▄▀░░░░░░░░░░█
█░░▄▀░░██░░▄▀░░█░░▄▀░░█████████░░▄▀░░░░▄▀░░░░██████░░▄▀░░░░░░▄▀░░░░░░▄▀░░█░░▄▀░░█████████
█░░▄▀░░░░░░▄▀░░█░░▄▀░░░░░░░░░░█░░▄▀░░▄▀░░░░░░██████░░▄▀░░██░░▄▀░░██░░▄▀░░█░░▄▀░░░░░░░░░░█
█░░▄▀▄▀▄▀▄▀▄▀░░█░░▄▀▄▀▄▀▄▀▄▀░░█░░▄▀▄▀▄▀▄▀▄▀░░██████░░▄▀░░██░░▄▀░░██░░▄▀░░█░░▄▀▄▀▄▀▄▀▄▀░░█
█░░▄▀░░░░░░▄▀░░█░░░░░░░░░░▄▀░░█░░▄▀░░░░░░▄▀░░██████░░▄▀░░██░░░░░░██░░▄▀░░█░░▄▀░░░░░░░░░░█
█░░▄▀░░██░░▄▀░░█████████░░▄▀░░█░░▄▀░░██░░▄▀░░██████░░▄▀░░██████████░░▄▀░░█░░▄▀░░█████████
█░░▄▀░░██░░▄▀░░█░░░░░░░░░░▄▀░░█░░▄▀░░██░░▄▀░░░░████░░▄▀░░██████████░░▄▀░░█░░▄▀░░░░░░░░░░█
█░░▄▀░░██░░▄▀░░█░░▄▀▄▀▄▀▄▀▄▀░░█░░▄▀░░██░░▄▀▄▀░░████░░▄▀░░██████████░░▄▀░░█░░▄▀▄▀▄▀▄▀▄▀░░█
█░░░░░░██░░░░░░█░░░░░░░░░░░░░░█░░░░░░██░░░░░░░░████░░░░░░██████████░░░░░░█░░░░░░░░░░░░░░█
█████████████████████████████████████████████████████████████████████████████████████████
████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████
█░░░░░░░░░░░░░░█░░░░░░██████████░░░░░░█░░░░░░░░██░░░░░░░░█░░░░░░░░░░░░░░█░░░░░░██░░░░░░█░░░░░░░░░░█░░░░░░██████████░░░░░░█░░░░░░░░░░░░░░█░░░░░░░░░░░░░░█
█░░▄▀▄▀▄▀▄▀▄▀░░█░░▄▀░░░░░░░░░░██░░▄▀░░█░░▄▀▄▀░░██░░▄▀▄▀░░█░░▄▀▄▀▄▀▄▀▄▀░░█░░▄▀░░██░░▄▀░░█░░▄▀▄▀▄▀░░█░░▄▀▄░░░░░░░░░██░░▄▀░░█░░▄▀▄▀▄▀▄▀▄▀░░█░░▄▀▄▀▄▀▄▀▄▀░░█
█░░▄▀░░░░░░▄▀░░█░░▄▀▄▀░░░░░░░░██░░▄▀░░█░░░░▄▀░░██░░▄▀░░░░█░░░░░░▄▀░░░░░░█░░▄▀░░██░░▄▀░░█░░░░▄▀░░░░█░░▄▀▄▀▄░░░░░░░██░░▄▀░░█░░▄▀░░░░░░░░░░█░░▄▀░░░░░░▄▀░░█
█░░▄▀░░██░░▄▀░░█░░▄▀░▄▀░░░░░░░██░░▄▀░░███░░▄▀▄▀░░▄▀▄▀░░███████░░▄▀░░█████░░▄▀░░██░░▄▀░░███░░▄▀░░███░░▄▀░░▄▀░░░░░░██░░▄▀░░█░░▄▀░░█████████░░░░░░██░░▄▀░░█
█░░▄▀░░░░░░▄▀░░█░░▄▀░░░▄▀░░░░░██░░▄▀░░███░░░░▄▀▄▀▄▀░░░░███████░░▄▀░░█████░░▄▀░░░░░░▄▀░░███░░▄▀░░███░░▄▀░░░▄▀░░░░░██░░▄▀░░█░░▄▀░░███████████████░░░░▄▀░░█
█░░▄▀▄▀▄▀▄▀▄▀░░█░░▄▀░░░░▄▀░░░░██░░▄▀░░█████░░░░▄▀░░░░█████████░░▄▀░░█████░░▄▀▄▀▄▀▄▀▄▀░░███░░▄▀░░███░░▄▀░░░░░▄▀░░░██░░▄▀░░█░░▄▀░░██░░░░░░███████░░▄▀░░░░█
█░░▄▀░░░░░░▄▀░░█░░▄▀░░░░░░▄▀░░░░░░▄▀░░███████░░▄▀░░███████████░░▄▀░░█████░░▄▀░░░░░░▄▀░░███░░▄▀░░███░░▄▀░░██░░░▄▀░░░░░▄▀░░█░░▄▀░░██░░▄▀░░███████░░░░░░███
█░░▄▀░░██░░▄▀░░█░░▄▀░░██░░░░▄▀░░░░▄▀░░███████░░▄▀░░███████████░░▄▀░░█████░░▄▀░░██░░▄▀░░███░░▄▀░░███░░▄▀░░██░░░░░▄▀░░░▄▀░░█░░▄▀░░██░░▄▀░░████████████████
█░░▄▀░░██░░▄▀░░█░░▄▀░░██░░░░░░▄▀░░▄▀░░███████░░▄▀░░███████████░░▄▀░░█████░░▄▀░░██░░▄▀░░█░░░░▄▀░░░░█░░▄▀░░██░░░░░░░▄▀░▄▀░░█░░▄▀░░░░░░▄▀░░███████░░░░░░███
█░░▄▀░░██░░▄▀░░█░░▄▀░░██░░░░░░░░▄▀▄▀░░███████░░▄▀░░███████████░░▄▀░░█████░░▄▀░░██░░▄▀░░█░░▄▀▄▀▄▀░░█░░▄▀░░██░░░░░░░░░░▄▀░░█░░▄▀▄▀▄▀▄▀▄▀░░███████░░▄▀░░███
█░░░░░░██░░░░░░█░░░░░░██████████░░░░░░███████░░░░░░███████████░░░░░░█████░░░░░░██░░░░░░█░░░░░░░░░░█░░░░░░██████████░░░░░░█░░░░░░░░░░░░░░███████░░░░░░███
████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████
    
```
# Feel free to ask your doubts 😉...

**Comment by Mubin19** [comment]
Hello, I am Mubin,
I am searching for a final year project,
I have seen your Agroworld project can you please provide me the admin side and firebase access of this project...

**Reply by theashishgavade** [reply]
It's My Major Project For Final year

I am really sorry, I can't provide Administrative access to anybody But i'll help you to run the project.

simply follow all the steps Listed in my GitHub to import Project Files

after that You have to take Fresh Mail where You can create "AgroWorld" Project I think you have to take some different name because Firebase doesn't allow to take similar Package name so you have to make changes according your new package name in whole code

after that you have to create two Types of DB in firebase 1 For Images and 1 for Real Time info/textual info

product
vehicle
Create these 2 Folders with same name in firebase
![image](https://github.com/theashishgavade/theashishgavade/assets/68729505/77b86323-2098-47d1-a5ea-e9bc8ac0e77e)

and  in Real Time DB You have to create 3 (Just Copy paste these 3 Localized for Hindi language and normal for english)
Localized_product
product
vehicle


Use these names as it is and after that you just have to connect Your Firebase to Your Code using API ID'S


after that you have to create 1 razor pay Student account for Trial Payments in application then you just have to connect account to project using API Api id


that's all for Creating Firebase and Razor pay gateway




for adding the products / Vehicles
In code I created IF else Statement where Transport and Manufacturers are listed just change these mails to Yours then you will get access to Transport and Manufacturer Pannels






I'll add articles sheets as soon as possible in same Repo
for these articles you have to copy and paste Excel File data to Google Excel File with same 



after that use sheet Db website to create api credentials for these files

after that change Api credentials in main code according yours

**Comment by Nikitakumari225138** [comment]
can you share your instagram id plz
